py-polars/polars/expr/string.py

```diff
@@ -658,6 +669,20 @@ def strip_chars_start(self, characters: IntoExprColumn | None = None) -> Expr:
         │ rld     │
         └─────────┘
 
+        The order of the provided characters does not matter, they behave like a set.
+
+        >>> pl.DataFrame({"foo": ["aabcdef"]}).select(
+        ...     pl.col("foo").str.strip_chars_start("cba")
+        ... )
```

Comment:
unrelated to this PR, but I think all of these examples would be clearer if there was a new column created, so the input and output are shown side-by-side - want to do that in a follow-up PR?

Reply:
Good point, will do! Then let's merge this for now.

Reply:
sure, just leaving open a bit in case anyone has objections